WinUIGallery/ControlPages/DesignGuidance/IconographyPage.xaml

```diff
@@ -206,6 +204,19 @@
                         <controls:SampleCodePresenter x:Name="XAMLCodePresenter" Style="{StaticResource CodeValuePresenterStyle}" />
                         <TextBlock Text="C#" />
                         <controls:SampleCodePresenter x:Name="CSharpCodePresenter" Style="{StaticResource CodeValuePresenterStyle}" />
+                        <TextBlock Text="Tags" />
+                        <ItemsControl ItemsSource="{Binding SelectedItem.Tags, Mode=OneWay}" Margin="0,4,0,4">
+                            <ItemsControl.ItemsPanel>
```

Comment:
Would it make sense, for a11y purposes, to use an e.g. `ItemsView` instead?

Reply:
Thanks! I just changed it to `ItemsView`.